src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -5372,6 +5379,9 @@ public void ManifestError(string msg, bool runtimeCritical = false)
                 throw new ArgumentException(msg);
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2075:UnrecognizedReflectionPattern",
```

Comment:
Can we update the ILLink.Suppressions.xml file as we make these changes? That way the file stays up to date, and we can see what other ILLink warnings are left in this area.

https://github.com/dotnet/runtime/blob/3e1e1c15a73301e34c96b086764da7b7802514f9/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Suppressions.Shared.xml#L136-L141